src/components/UserProfile/UserSettings/UserGeneralSettings/index.tsx

```diff
@@ -186,21 +220,96 @@ const UserGeneralSettings = () => {
             <Form className="section">
               <div className="form-row">
                 <label className="text-label">
-                  {intl.formatMessage(messages.accounttype)}
+                  {intl.formatMessage(messages.connectedaccounts)}
                 </label>
-                <div className="mb-1 text-sm font-medium leading-5 text-gray-400 sm:mt-2">
-                  <div className="flex max-w-lg items-center">
-                    {user?.userType === UserType.PLEX ? (
-                      <Badge badgeType="warning">
-                        {intl.formatMessage(messages.plexuser)}
-                      </Badge>
+                {!currentSettings.plexLoginEnabled && user?.id !== 1 && (
+                  <div className="mb-1 text-sm font-medium leading-5 text-gray-400 sm:mt-2">
+                    <div className="flex max-w-lg items-center">
+                      {intl.formatMessage(messages.noconnectedavailable)}
+                    </div>
+                  </div>
+                )}
+                {(currentSettings.plexLoginEnabled || user?.id === 1) && (
+                  <div className="flex items-center rounded sm:col-span-2">
+                    <div className="mr-4 flex h-7 w-7 items-center justify-center rounded-full border border-gray-700 bg-gray-800">
+                      <CheckCircleIcon
+                        className={`h-full w-full ${
+                          user?.isPlexUser ? 'text-green-500' : 'text-gray-700'
+                        }`}
+                      />
+                    </div>
+                    <PlexLogo className="h-8 border-r border-gray-700 pr-4" />
+                    {user?.id !== currentUser?.id ? (
+                      <div className="ml-4 text-sm text-gray-400">
+                        {intl.formatMessage(messages.onlyloggedinuseredit)}
+                      </div>
                     ) : (
-                      <Badge badgeType="default">
-                        {intl.formatMessage(messages.localuser)}
-                      </Badge>
+                      <>
+                        {!user?.isPlexUser ? (
+                          <>
+                            <div className="ml-4">
+                              <LoginWithPlex
+                                onComplete={() => {
+                                  revalidateUser();
+                                }}
+                                textOverride={intl.formatMessage(
+                                  messages.connectplexaccount
+                                )}
+                              />
+                            </div>
```

Comment:
We may need to revalidate settings when updating plexLoginEnabled since it doesn't seem to reflect without a hard refresh.